osquery/tables/system/darwin/processes.cpp

```diff
@@ -291,6 +291,30 @@ void genProcUniquePid(QueryContext& context, int pid, Row& r) {
   }
 }
 
+void genProcArch(QueryContext& context, int pid, Row& r) {
+  if (!context.isColumnUsed("cpu_type") &&
+      !context.isColumnUsed("cpu_subtype")) {
+    return;
+  }
+
+  struct proc_uniqidentifierinfo {
+    cpu_type_t p_cputype;
+    cpu_subtype_t p_cpusubtype;
+  };
+
+  struct proc_uniqidentifierinfo proc_archinfo {
+    0, 0
+  };
+  int status = proc_pidinfo(pid, 19, 0, &proc_archinfo, sizeof(proc_archinfo));
```

Comment:
It's a bit unclear what `19` means here. It is a `flavor `, right? Can we have proper const or definition from system header file? Or at least leave a small comment here.